### DISCUSSION TITLE
Not most up to date version of Crashalytics

### DISCUSSION THREAD

**Discussion by hannahcx** [invertase/react-native-firebase#4262] [Q&A] [answered]
In the Firebase UI it says I must update to newest version of crashalytics, but when I update firebase via react native - the Pods do not match up with the new Pods required in the Firebase documentation (recently changed). It now needs to be pod 'Firebase/Analytics' for instance rather than 'Crashlytics' for ios.

**Comment by mikehardy** [comment]
You can see what software will actually be in use by inspecting our end-to-end testing harness https://github.com/invertase/react-native-firebase/tree/master/tests

**Comment by hannahcx** [comment]
No, I mean the react-native-firebase is not up to date, and needs to be updated to match the current firebase docs, when is this work due in the pipeline?

**Reply by hannahcx** [reply]
Firebase/Crashalytics Pod will no longer work in Novemeber, and you guys need to update to just 'Crashalytics'

**Comment by mikehardy** [comment, marked as answer]
1. This is open source, when I see "when is this work due in the pipeline" and "you guys need to...", I think, great, when are you going to contribute? We welcome PRs!
2. #3580 - it's done, consult the details of our tests app, dig into the docs for real, and you should be able to verify the same

Cheers
